### ISSUE TITLE
can't install skype.

### ISSUE THREAD

**Issue by castilma** [NixOS/nixpkgs#31047]
## Issue description
when trying to install skype 5.4 with `nix-env -iA nixos.skype` the download fails with a 404.
[indeed](https://repo.skype.com/deb/pool/main/s/skypeforlinux/), the requested file is no longer there.
this package needds to be updated.

as a temporary workaround, one could add this to .config/nixpkgs/config.nix:
```
{
...
    myskype = pkgs.skype.overrideAttrs (oldAttrs: rec {
      src = pkgs.fetchurl {
        url = "https://repo.skype.com/deb/pool/main/s/skypeforlinux/skypeforlinux_5.5.0.1_amd64.deb";
        sha256 = "1r65qlsjwp0n0fmlhvbp71h03b3x73r26jk9f4q687sjqnzmkvcr"; };
     });
...
}
```
and install through `nix-env -iA nixos.myskype`

another question would be, why we are using 5.4 if there is already 8.10?

## Technical details

* System: 17.03.1938.630dbfe672 (Gorilla)
* Nix version: nix-env (Nix) 1.11.14
* Nixpkgs version: "17.09.1905.a0b50eab59"

**Comment by orivej** [comment]
Released in 0dbd1b8b5df280c670f38baa29b68373544a9638.

> why we are using 5.4 if there is already 8.10?

I don't know, but one reason not to rush with 8.* is because any particular version may disappear very soon: the oldest available deb is just two weeks old https://repo.skype.com/deb/pool/main/s/skypeforlinux/

**Comment by vcunat** [comment]
Confirming the fix.
